File: official/docs/csharp/v4/referral-customers/refund-by-payment-log.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            PaymentRefund refund = await Client.Beta.Referral.RefundByPaymentLog("paylog_...");

            Console.WriteLine(JsonConvert.SerializeObject(refund, Formatting.Indented));
        }
    }
}

File: official/docs/golang/v4/referral-customers/refund-by-payment-log.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func refundByPaymentLog() {
	client := easypost.New("EASYPOST_API_KEY")

	refund, _ := client.BetaRefundByPaymentLog("paylog_...")

	fmt.Println(refund)
}
